pokedex/model/Makefile

```diff
@@ -36,51 +31,87 @@ $(PREFIX)/lib $(PREFIX)/include: $(BUILD_DIR)/2-cgen/makefile
 # ---------------------------------------
 # Stage 0: generate ASL for code snippets
 # ---------------------------------------
-$(BUILD_DIR)/0-aslgen/$(FINISH_MARKER): $(CODEGEN_ASL_SRCS) $(RISCV_OPCODES_SRC)
-	mkdir -p $(@D)
-	$(CODEGEN_CLI) \
+STAGE_0 := $(BUILD_DIR)/0-aslgen
+RVV_INSTS := vmax_vv.asl vmax_vx.asl vmin_vv.asl \
+			 vmin_vx.asl vmaxu_vv.asl vmaxu_vx.asl \
+			 vminu_vv.asl vminu_vx.asl
```

Comment:
is there anything vmax/vmin different from others?

Reply:
No, I just copy from previous build script.

Reply:
It seems all other toml generated rvv instructions are not included. Requires adding mall toml-generated instructions manually here?